src/components/html.cjsx

```diff
@@ -35,7 +36,7 @@ module.exports = React.createClass
   getHTMLFromProp: ->
     {html} = @props
     if html
-      __html: html
+      __html: wrapFrames(html)
```

Comment:
It seems this constructs (and discards) a DOM node every time render is called (which react's diffing uses to compute whether to update the DOM), rather than just when the DOM changes. Would it be better to wrap the iFrame on `componentDidMount`?